Library/Homebrew/cli/parser.rb

```diff
@@ -406,11 +425,12 @@ def generate_help_text
                  end
       end
 
+      sig { void }
       def cask_options
         self.class.global_cask_options.each do |args|
-          options = args.pop
+          options = T.cast(args.pop, T::Hash[Symbol, String])
           send(*args, **options)
-          conflicts "--formula", args.last
+          conflicts "--formula", args[1]
```

Comment:
The type of `last` has changed due to the `pop` above, but sorbet isn't aware of this, so we access it by index instead